Module/src/PSWordCloud/NewWordCloudCommand.cs

```diff
@@ -809,6 +808,53 @@ protected override void EndProcessing()
 
         #region HelperMethods
 
+        private IEnumerable<string> NormalizeInput(PSObject input)
+        {
+            if (MyInvocation.ExpectingInput)
+            {
+                if (input.BaseObject is string s)
+                {
+                    yield return s;
+                }
+                else
+                {
+                    yield return LanguagePrimitives.ConvertTo<string>(input);
+                }
+            }
+            else
+            {
+                switch (input.BaseObject)
+                {
+                    case string[] sa:
+                        foreach (var line in sa)
+                        {
+                            yield return line;
+                        }
+
+                        yield break;
+
+                    case string s2:
+                        yield return s2;
+                        yield break;
+
+                    default:
+                        var enumerable = LanguagePrimitives.GetEnumerable(input.BaseObject);
```

Comment:
`GetEnumerable` can throw iirc (like with `default(ImmutableArray<string>)`).

Reply:
oh, lovely... good to know lol